File: Lua/05_functions.lua
-- 05_functions.lua
local function add(a, b)
    return a + b
end

local function greet(name, age)
    age = age or 25
    print("Hello " .. name .. ", you are " .. age)
end

greet("Bob")
greet("Alice", 32)
print("Sum: " .. add(5, 7))

File: Java/05_functions.java
public class Functions {
    static int add(int a, int b) {
        return a + b;
    }

    static void greet(String name, int age) {
        System.out.println("Hello " + name + ", you are " + age);
    }

    public static void main(String[] args) {
        greet("Bob", 25);
        greet("Alice", 32);
        System.out.println("Sum: " + add(5, 7));
    }
}
